index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly remove disabled linters as linters help us to write a cleaner code

Reply:
I'll work on it thanks for the feedback! ❤️

Reply:
Addressed 🔥!